packages/itwinui-react/src/core/Carousel/CarouselSlider.tsx

```diff
@@ -8,7 +8,7 @@ import { CarouselContext } from './CarouselContext.js';
 import {
   getWindow,
   useMergedRefs,
-  useIsomorphicLayoutEffect,
+  useIsomorphicLayoutEffect as useLayoutEffect,
```

Comment:
Just thinking out loud: What is better, importing as an alias every time; or naming the export itself as `useLayoutEffect` and then no import aliases are required?

Reply:
It introduces potential confusion about `React.useLayoutEffect` vs `useLayoutEffect`. Ideally we'd have a custom lint rule that that flags usages of `React.useLayoutEffect`. 

For now, I'm ok with renaming the export to `useLayoutEffect`. 👍

Make sure to update the STYLEGUIDE after that.

Reply:
Latest commit renames the export to `useLayoutEffect`.